cli/tools/vendor/import_map.rs

```diff
@@ -205,7 +205,10 @@ pub fn build_import_map(
       if let Ok(resolved_url) =
         import_map.resolve(&specifier_text, import_map.base_url())
       {
-        builder.imports.add(specifier_text, &resolved_url);
+        // the graph might not have any jsx files
+        if graph.contains(&resolved_url) {
```

Comment:
Actually, perhaps it should still vendor the jsx import source in this case? I will mark this PR back as draft.

Reply:
Ok, this now always vendors the jsx import source.